src/main/java/nextstep/subway/line/domain/Sections.java

```diff
@@ -0,0 +1,82 @@
+package nextstep.subway.line.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import nextstep.subway.station.domain.Station;
+
+@Embeddable
+public class Sections {
+    private static final int MIN_SIZE = 1;
+
+    @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
+    private List<Section> sections;
+
+    protected Sections () {
+        this.sections = new ArrayList<>();
+    }
```

Comment:
사소하지만 명시적 초기화를 해주는 것이 JPA 컨벤션으로 알고 있습니다 ! 😄 

```suggestion
    private List<Section> sections = new ArrayList<>();

    protected Sections () {
    }
```